code/core/assets/server/base-preview-body.html

```diff
@@ -114,6 +114,6 @@ <h1 id="error-message"></h1>
         >.
       </li>
     </ol>
-    <pre class="sb-errordisplay_code"><code id="error-stack"></code></pre>
+    <pre class="sb-errordisplay_code chromatic-ignore"><code id="error-stack"></code></pre>
```

Comment:
we should ideally do this dynamically in our test storybooks otherwise this can actually affect Chromatic users, couldn't it?

Reply:
Here's my proposal:
```ts
export const MissingStory = {
  render: () => (
    <IFrame
      active
      id="iframe"
      title="Missing"
      src="/iframe.html?id=missing"
      allowFullScreen
      style={style}
      scale={1.0}
    />
  ),
  play: async () => {
    await waitFor(
      () => {
        const iframeBody = (document.querySelector('#storybook-root iframe') as HTMLIFrameElement)?.contentWindow
          ?.document
        const displayError = (iframeBody as Document).querySelector('.sb-errordisplay_code');
        if (displayError !== null) {
          displayError?.classList.add('chromatic-ignore');
          return displayError;
        }
        throw new Error('Retrigger waitfor');
      },
      { timeout: 5000 }
    );
  },
};
```


I don't know if this will work because the story is rendering inside of an iframe. I don't know whether chromatic considers ignore regions coming from within iframes inside of iframes 🤔